packages/main/src/themes/sap_horizon_dark/SplitButton-parameters.css

```diff
@@ -16,6 +16,13 @@
 	--_ui5_split_button_inner_focused_border_radius_outer: 0.375rem;
 	--_ui5_split_button_inner_focused_border_radius_inner: 0.375rem;
 
+	/* Separator colors */
+	--_ui5_split_button_emphasized_separator_color: transparent;
+	--_ui5_split_button_positive_separator_color: transparent;
+	--_ui5_split_button_negative_separator_color: transparent;
+	--_ui5_split_button_attention_separator_color: transparent;
```

Comment:
In several theme specific parameters file, this parameter is defined twice, the second will nevertheless be applied, remove the first occurrence 
<img width="627" alt="Screenshot 2023-11-26 at 20 57 22" src="https://github.com/SAP/ui5-webcomponents/assets/15702139/8b0faa2c-1489-489f-b526-5be05bb6806b">